scripts/summary.py

```diff
@@ -7,7 +7,7 @@
 import prettytable
 
 
-DEFAULT = ['glam', 'cgmath', 'nalgebra', 'euclid', 'vek']
+DEFAULT = ['glam', 'cgmath', 'nalgebra', 'euclid', 'vek', 'static-math']
```

Comment:
Should remove from `DEFAULT` and add to `OPTIONAL`

Reply:
Ok